rest/rest-client/src/test/resources/functionaltests/descriptors/add_and_remove_external_endpoint_urls.groovy

```diff
@@ -0,0 +1,14 @@
+schedulerapi.connect()
+
+schedulerapi.addExternalEndpointUrl(variables.get("PA_JOB_ID"), "http://aaa.fr")
+schedulerapi.addExternalEndpointUrl(variables.get("PA_JOB_ID"), "http://bbb.fr")
+
+schedulerapi.removeExternalEndpointUrl(variables.get("PA_JOB_ID"), "http://aaa.fr")
+
+schedulerapi.addExternalEndpointUrl(variables.get("PA_JOB_ID"), "http://ccc.fr")
+schedulerapi.addExternalEndpointUrl(variables.get("PA_JOB_ID"), "http://ddd.fr")
+
+schedulerapi.removeExternalEndpointUrl(variables.get("PA_JOB_ID"), "http://aaa.fr")
```

Comment:
it seems that you are removing an endpoint which was already removed, what do you expect to have here (an exception ?)

Reply:
To be sure it does not crash.